GoogleChromeLabs/ps-analysis-tool
# Feature: Use Floating UI for tooltip positioning

## Description
Till now the position of the tooltip was being calculated manually which was very intensive and was bound to not cover all the edge cases. This PR aims to introduce a library which will fix the tooltip positioning.

## Relevant Technical Choices
- Remove manual calculation of the positioning.
- Use the Floating UI library to calculate the position.

## Testing Instructions
- In the terminal run `npm start`.
- Use the frame overlay feature in PS-Analysis-Tool tab.
- You should see the tooltip positioning has become better than previous versions.


Fixes #219
